lib/utilities/default-targets.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
+  browsers: [
+    'ie 9',
+    'last 1 Chrome versions',
```

Comment:
No argument there. In this PR I'm just taking advantage of the new targets, so I refactored to a separate util to avoid duplication.